### PR TITLE
Features/ntr 467/add docs to services

### PR DESCRIPTION
**Pull request by likeuwill**
## Motivation and Context

[JIRA](https://eaflood.atlassian.net/browse/NTR-467)
## Description

I also moved `SES_FROM_EMAIL` variable to app.rb because we use it for all project configuration. I think it's a good idea to hold all settings in one place.
## Checklist:

- [ ] It contains only changes required by issue (does not contain other PR)
- [ ] Includes a link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -22,10 +22,13 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('lib')
 
-    config.x.session_timeout_in_min = (ENV['SESSION_TIMEOUT'].presence || 15).to_i
-
-    feedback_url_default = 'https://www.surveymonkey.co.uk/r/2T8BX2D'
-    config.x.feedback_url = (ENV['FEEDBACK_URL'].presence || feedback_url_default)
+    # timeout the user session without activity.
+    config.x.session_timeout_in_min = ENV.fetch('SESSION_TIMEOUT', 15)
```
Comment (miqs1992) [inline config/application.rb]: `ENV.fetch('SESSION_TIMEOUT', 15).to_i`

File: app/services/csv_upload_service.rb [hunk 2]
```diff
@@ -1,47 +1,89 @@
...
+  # Attributes used internally to save values.
   attr_reader :file, :error, :user
 
+  ##
+  # Initializer method.
+  #
+  # ==== Attributes
+  #
+  # * +file+ - object
+  # * +user+ - object
```
Comment (miqs1992) [inline app/services/csv_upload_service.rb]: an instance of the User class

File: app/services/csv_upload_service.rb [hunk 3]
```diff
@@ -1,47 +1,89 @@
...
   UPLOAD_ERROR_MSG = 'The selected file could not be uploaded – try again'
+  # Attributes used internally to save values.
   attr_reader :file, :error, :user
 
+  ##
+  # Initializer method.
+  #
+  # ==== Attributes
+  #
+  # * +file+ - object
```
Comment (miqs1992) [inline app/services/csv_upload_service.rb]: uploaded file

File: app/services/email_validator.rb [hunk 4]
```diff
@@ -1,12 +1,25 @@
 # frozen_string_literal: true
 
+##
+# This class is used to validate email address.
+#
 class EmailValidator < BaseService
+  # Attribute used internally to save values.
   attr_reader :email
```
Comment (miqs1992) [inline app/services/email_validator.rb]: can this be moved to private?

File: config/application.rb (hunk 1 above)
Comment (likeuwill) [reply config/application.rb]:
```
[1] pry(main)> ENV.fetch('SESSION_TIMEOUT', 15)
=> 15
[2] pry(main)> ENV.fetch('SESSION_TIMEOUT', 15).class
=> Integer
[3] pry(main)> 
```

Comment (miqs1992) [reply config/application.rb]: Do you have 'SESSION_TIMEOUT' set? ENV always returns a string

Comment (likeuwill) [reply config/application.rb]: You right, fixed

**Pull request merged**